src/eyeos-rabbitmq-auth-backend-http.js

```diff
@@ -37,17 +37,28 @@ var logger = log2out.getLogger('rabbitmq-auth-backend-http');
 
 logger.info('Current settings are:\n', settings);
 
-if (cluster.isMaster) {
-    // Fork workers.
-    for (var i = 0; i < numCPUs; i++) {
-        cluster.fork();
+var rabbitmqAuhBackendHttp, notifier;
```

Comment:
I think these variables don't need to be global, they can be declared in the `startApplication` function (in fact, the `notifier` variable is being declared there too).